### PR TITLE
SMHE-590 Add validation and tests for Set activity calendar

### PR DESCRIPTION
**Pull request by stefanermens**

### CODE REVIEW

**Comment by stefanermens** [comment]
Retest this please

**Comment by Marcel-Jansen** [comment]
Retest this please

**Comment by stefanermens** [comment]
Retest this please

**Comment by Marcel-Jansen** [comment]
Retest this please

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/ActivityCalendarValidatorTest.java [hunk 1]
```diff
@@ -186,6 +191,20 @@ void testMultipleUsesOfSameWeek() {
                 this.createActivityCalendarDto(TestAnomaly.MULTIPLE_USES_SAME_WEEK)));
   }
 
+  @Test
+  void testInvalidSeasonName() {
+    final byte[] test = new byte[] {"0".getBytes()[0]};
```
Comment (harrymiddelburgetall) [inline osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/ActivityCalendarValidatorTest.java:196]: How does variable _test_ contribute in this test?

**Review by harrymiddelburgetall — CHANGES_REQUESTED** [review]

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/ActivityCalendarValidatorTest.java (hunk 1 above)
Comment (stefanermens) [reply osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/ActivityCalendarValidatorTest.java:196]: It's some testcode I forgot to remove.

**Review by harrymiddelburgetall — COMMENTED** [review]
Not to block this PR.

**Review by harrymiddelburgetall — APPROVED** [review]

**Pull request merged**